lib/azure/resource_management/ARM_interface.rb

```diff
@@ -139,9 +139,8 @@ def delete_server(resource_group_name, vm_name)
           ui.info 'Deleting ..'
 
           begin
-            print '.'
-            promise = compute_management_client.virtual_machines.delete(resource_group_name, vm_name)
-          end until promise.value!.body.nil?
+            server_detail = compute_management_client.virtual_machines.delete(resource_group_name, vm_name)
+          end until server_detail.value!.body.nil?

```

Comment:
Does `server_detail` still has `.value!.body` method in the latest version of `azure-sdk-for-ruby`?
If yes then please ignore this comment.